dotnet/src/Connectors/Connectors.Memory.Chroma/ChromaMemoryStore.cs

```diff
@@ -70,10 +71,10 @@ public async Task DeleteCollectionAsync(string collectionName, CancellationToken
         {
             await this._chromaClient.DeleteCollectionAsync(collectionName, cancellationToken).ConfigureAwait(false);
         }
-        catch (ChromaClientException e) when (e.CollectionDoesNotExistException(collectionName))
+        catch (SKException e) when (e.Message.Contains(string.Format(CultureInfo.InvariantCulture, CollectionDoesNotExistErrorFormat, collectionName)))
```

Comment:
nit: I would place this long condition in separate method since it is used in two places in this class. It will also improve the readability of `catch` block.

Reply:
Yep, good idea, done.